src/take.js

```diff
@@ -20,13 +23,21 @@ var slice = require('./slice');
  * @return {Array}
  * @example
  *
- *      R.take(3,[1,2,3,4,5]); //=> [1,2,3]
+ *      R.take(1, ['foo', 'bar', 'baz']); //=> ['foo']
+ *      R.take(2, ['foo', 'bar', 'baz']); //=> ['foo', 'bar']
+ *      R.take(3, ['foo', 'bar', 'baz']); //=> ['foo', 'bar', 'baz']
+ *      R.take(4, ['foo', 'bar', 'baz']); //=> ['foo', 'bar', 'baz']
+ *      R.take(3, 'ramda');               //=> 'ram'
  *
- *      var members= [ "Paul Desmond","Bob Bates","Joe Dodge","Ron Crotty","Lloyd Davis","Joe Morello","Norman Bates",
- *                     "Eugene Wright","Gerry Mulligan","Jack Six","Alan Dawson","Darius Brubeck","Chris Brubeck",
- *                     "Dan Brubeck","Bobby Militello","Michael Moore","Randy Jones"];
+ *      var members = [
+ *        'Paul Desmond', 'Bob Bates', 'Joe Dodge', 'Ron Crotty',
+ *        'Lloyd Davis', 'Joe Morello', 'Norman Bates', 'Eugene Wright',
+ *        'Gerry Mulligan', 'Jack Six', 'Alan Dawson', 'Darius Brubeck',
+ *        'Chris Brubeck', 'Dan Brubeck', 'Bobby Militello', 'Michael Moore',
+ *        'Randy Jones'
+ *      ];
```

Comment:
My previous comment had a an appropriately abbreviated list, duplicated below.

[Take Five][tf] is one of the most famous jazz recordings ever, off Time Out, performed by the Dave Brubeck Quartet (unfortunately a quartet!)  The original list seems to be every person who was ever a member of that quartet (except, amusingly enough, the front man himself.)  The list below includes the four who were on that album and a few others just to fill it out.

```js
    var takeFive = R.take(5);
    var personnel = ['Dave Brubeck', 'Paul Desmond', 'Eugene Wright', 'Joe Morello', 
                     'Gerry Mulligan', 'Bob Bates', 'Joe Dodge', 'Ron Crotty'];
    var lineup = takeFive(personnel); 
    //=> ['Dave Brubeck', 'Paul Desmond', 'Eugene Wright', 'Joe Morello', 'Gerry Mulligan']
```

(Both the album and the song almost always one of the top few in the Top N jazz lists, often beat out by only Miles Davis and John Coletrane.  It was the first jazz record to sell a million copies.)

  [tf]: https://www.youtube.com/watch?v=vmDDOFXSgAs